Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -451,3 +451,6 @@ uplink-combat-bakery-desc = A kit of clandestine baked weapons. Contains a bague
 
 uplink-business-card-name = Syndicate Business Card
 uplink-business-card-desc = A business card that you can give to someone to demonstrate your involvement in the syndicate or leave at the crime scene in order to make fun of the detective. You can buy no more than three of them.
+
+uplink-smuggler-satchel-name = Smugglers Satchel
+uplink-smuggler-satchel-desc = A satchel that can fit under station tiles.
```

Comment:
These loc strings should be introduced in `Resources/Locale/en-US/_Harmony/store/uplink-catalog.ftl` instead.

Reply:
Additionally, `
uplink-smuggler-satchel-name` should be lowercase to match every other item name in the game.